RecoParticleFlow/PFProducer/python/particleFlowEGamma_cff.py

```diff
@@ -23,14 +27,29 @@
 particleBasedIsolationTmp.valueMapElePFblockIso = cms.string("gedGsfElectronsTmp")
 
 egmPhotonIsolationCITK = _egmPhotonIsolationAOD.clone()
+egmElectronIsolationCITK = _egmElectronIsolationCITK.clone()
+egmElectronIsolationPileUpCITK = _egmElectronIsolationCITKPileUp.clone()
 
-for iPSet in IsoConeDefinitionsTmp:
+#change particleBasedIsolation object to tmp
+for iPSet in IsoConeDefinitionsPhotonsTmp:
```

Comment:
yes, it worked. Can you explain for my education why .copy() is preferred? I got the impression that one has to be very careful with doing copies in python